content/Cost/200_Labs/200_6_Cost_Anomaly_Detection/4_create_individual_alert.md

```diff
@@ -40,28 +43,46 @@ The default SNS access policy allow all Principals with in the owner account to
   "Resource": "your topic ARN"
 }
 ```
+========================================================
 
-5. Once the SNS topic is successfully created, a subscription needs to be created. Under the **Subscription** tab On the SNS topic page, click **Create Subscription**:
+5. Create a subscription once the SNS topic is successfully created. Under the **Subscriptions** tab, click **Create Subscription**:
 ![Images/CostAnomaly21.png](/Cost/200_6_Cost_Anomaly_Detection/Images/cost_anomaly_21.png?classes=lab_picture_small)
 
-6. On the **Create Subscription** page, select **Email** under the **Protocol** drop down menu and enter an email address in the text field under **Endpoint**. Then click **Create Subscription** at the bottom of the page to finish creating subscription:
+6. On the **Create Subscription** page, select **Email** under the **Protocol** drop-down menu and enter an email address in the text field under **Endpoint**. Then click **Create Subscription** at the bottom of the page to finish creating subscription:
 ![Images/CostAnomaly22.png](/Cost/200_6_Cost_Anomaly_Detection/Images/cost_anomaly_22.png?classes=lab_picture_small)
 
-At times, there is a requirement to alert multiple individuals or multiple teams such as FinOps, DevOps, SRE or Management team. In such a scenario additional subscriptions can be created.
+At times, there is a requirement to alert multiple individuals or multiple teams such as FinOps, DevOps, SRE or Management team. In such a scenario, additional subscriptions can be created.
+
+{{% notice note %}}
+To send notifications, you must accept the subscription to the Amazon SNS notification topic.
+{{% /notice %}}
+
+7. Take a note of Amazon SNS ARN: 
+![Images/CostAnomaly22-ARN.png](/Cost/200_6_Cost_Anomaly_Detection/Images/cost_anomaly_22-arn.png?classes=lab_picture_small)
 
 {{% notice note %}}
-For notifications to be sent, you must accept the subscription to the Amazon SNS notification topic.
+Amazon SNS ARN is required to complete the rest of steps. 
 {{% /notice %}}
 
-7. Now go back to Cost Anomaly Detection **Create a subscription** page and enter SNS ARN:
+8. Now go back to the **Create a subscription** page where you left off at step 2 in Cost Anomaly Detection and enter SNS ARN you recorded:
 ![Images/CostAnomaly23.png](/Cost/200_6_Cost_Anomaly_Detection/Images/cost_anomaly_23.png?classes=lab_picture_small) 
 
-8. Next select a **Threshold** type and enter the threshold value. As an example, we have selected an absolute threshold and set the value to 100. Under **Cost monitors**, select you pre-existing cost monitor and then click **Create subscription**. 
+**[Janghan] What could be the reason we set the threshold to 100? we set it to 20 in Lab 2.** 
```

Comment:
In section 2, we selected percentage threshold. So setting a value of 20 was implicated 20% increase in spend. In this section, we are selecting absolute threshold. Setting a value of 100 means $100 increase in spend. Having said that, both 20% and $100 are hypothetical values. Customer has to choose these thresholds which makes sense depending on their overall spend and fluctuation in spend which are considered abnormal.